helm/robusta/values.yaml

```diff
@@ -540,7 +543,7 @@ kube-prometheus-stack:
         - name: 'null'
         - name:  'robusta'
           webhook_configs:
-            - url: 'http://robusta-runner.{{ .Release.Namespace }}.svc.cluster.local/api/alerts'
+            - url: 'http://robusta-runner.{{ .Release.Namespace }}.svc.{{ .Values.global.clusterDomain }}/api/alerts'
```

Comment:
The url you'd like on your case is `http://robusta-runner.default/api/alerts` right?
What will you put in `clusterDomain` for your cluster? is it ""
I think that will result in `http://robusta-runner.default.svc./api/alerts` 
Is it working this way?

Reply:
In my case, I'll set it to my GKE cluster's custom suffix. So it will still use an FQDN instead of an unqualified name.